common/src/main/java/com/revenuecat/purchases/common/networking/RCHTTPStatusCodes.kt

```diff
@@ -10,4 +10,5 @@ object RCHTTPStatusCodes {
     const val ERROR = 500
 
     fun isSuccessful(statusCode: Int) = statusCode < BAD_REQUEST
+    fun isServerError(statusCode: Int) = statusCode >= ERROR
```

Comment:
Are timeouts 500s in Android? Because I believe if the server is completely down that it timeouts, we want to compute the offline customer info, right?

Reply:
Hmm I think that would depend on how the server is configured right? I think you can configure how to respond at different levels in the server stack... Also, if the whole infrastructure was completely down, I guess we would have a reachability exception? In any case, I think this is something we should confirm with COIN 👍

Reply:
Also, I don't think we want to enable offline entitlements on a timeout, since it could possibly be a connection issue.

Reply:
Just checked the slack conversation